cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -6106,24 +6106,25 @@ alert.track = 0
 cron_schedule = 10 * * * *
 description = This report update the device inventory every hour and generates lookup for crowdstrike data. \
 \
-Data Collection - The crowdstrike data needs to be collected via the CrowdStrike Event Stream Add-on (https://splunkbase.splunk.com/app/5082/).
+Data Collection - The crowdstrike data needs to be collected via the CrowdStrike Falcon Devices Technical Add-On (https://splunkbase.splunk.com/app/5570).
 dispatch.earliest_time = -62m@m
 dispatch.latest_time = -2m@m
 display.general.type = statistics
 display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = `cs_crowdstrike_eventstream` | eval crowdstrike_userid=coalesce('event.UserId', 'event.UserName', user) | dedup crowdstrike_userid \
-| eval ip=lower(coalesce('event.UserIp', 'event.LocalIP', src)), hostname=lower(coalesce('event.ComputerName', 'event.HostnameField')), time=_time, indextime=_indextime, product_name="CrowdStrike", product_uuid=crowdstrike_userid \
-| rename host as crowdstrike_collected_by, "metadata.customerIDString" as crowdstrike_customer_id, "event.MACAddress" as mac_address \
+search = `cs_crowdstrike_devices` | dedup falcon_device.device_id \
+| eval time=_time, indextime=_indextime, product_name="CrowdStrike Devices" \
```

Comment:
For upgrade scenario, need to cleanup the device inventory lookup for product="CrowdStrike"

Also, on dashboard searches, need to update the product_names="CrowdStrike" to product_names="CrowdStrike Devices"